Add data income routine

rafaellehmkuhl opened the pull request.

joaoantoniocardoso commented on services/sats_comm/main.py:
```diff
@@ -45,6 +38,107 @@ def send_data_through_rockblock():
+
+def deal_with_income_data(income_data: bytes) -> None:
+    if income_data.decode().startswith("waypoint"):
+        _, lat, lon, wait_time, next_id = income_data.decode().split(":")
```
Note that this unpacking to 5 variables can fail if we have less or more than 4 ":", raising ValueError.

nevermind, I just saw that this function is being called from a try.

rafaellehmkuhl replied:
You mean if we send the wrong message?
Should we put a logger for the exception?

joaoantoniocardoso replied:
I think it is fine because the caller is using a try and is already debugging the exception =)

joaoantoniocardoso commented on services/sats_comm/main.py:
```diff
@@ -45,6 +38,107 @@ def send_data_through_rockblock():
+
+def init_rockblock() -> Tuple[RockBlock, serial.Serial]:
+    ser = serial.Serial(
+        "/dev/serial/by-path/platform-fd500000.pcie-pci-0000:01:00.0-usb-0:1.1:1.0-port0",
```
Can't we pass it as a CLI argument "--serial", like as in the other services?

rafaellehmkuhl replied:
Yup. Forgot to do it.
Fixed.

rafaellehmkuhl committed:
Saturate gathered values

rafaellehmkuhl committed:
Remove unused import

rafaellehmkuhl committed:
Add routine to deal with incoming data

rafaellehmkuhl committed:
Add data income routine

- Uses async logic to allow for concurrent data-income and data-outcome routines

rafaellehmkuhl committed:
Change default rest time for data outcome

rafaellehmkuhl committed:
Set serial por for satellite communication with CLI

rafaellehmkuhl committed:
Add satellite transfer method that resets the ring alert
